docs/reference/errors-and-warnings/NU1605.md

```diff
@@ -12,6 +12,8 @@ f1_keywords:
 
 # NuGet Warning NU1605
 
+[!INCLUDE [nugetsolver-tool](../../includes/nugetsolver-tool.md)]
```

Comment:
This is our highest hitting warning, while I do want to feature solver, I think the solver should still be provided as an alternative rather than the solution.

Reply:
There are already 4 solution sections in this doc. Should I place this one 4 times as alternative solution.
@aortiz-msft @JonDouglas What is your thought on this?

Reply:
I don't necessarily even know if I'd name nuget solver a solution in the traditional way, since *it upgrades* packages as well, but yeah, this is as much as of branding question as well, so I'd love to hear from the 2 of them.